client/my-sites/site-settings/site-visibility/launch-site.jsx

```diff
@@ -76,7 +77,11 @@ const LaunchSite = () => {
 	const price = formatCurrency( agency?.prices?.actual_price, agency?.prices?.currency );
 	const siteReferralActive = agency?.referral_status === 'active';
 	const shouldShowReferToClientButton =
-		isDevelopmentSite && ! siteReferralActive && ! agencyLoading;
+		config.isEnabled( 'a4a-dev-sites-referral' ) &&
+		isDevelopmentSite &&
+		! siteReferralActive &&
+		! agencyLoading;
+	const shouldShowAgencyBillingMessage = isDevelopmentSite && ! agencyLoading;
```

Comment:
Are we intentionally dropping the `! siteReferralActiv` condition from the AND statement?

Reply:
Huh? It wasn't dropped.

Reply:
Ah, you meant for `shouldShowAgencyBillingMessage` instead of `shouldShowReferToClientButton`. Good catch! Fixed!